src/main/java/org/opentripplanner/prediction_client/VehiclePresence.java

```diff
@@ -0,0 +1,184 @@
+package org.opentripplanner.prediction_client;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import org.locationtech.jts.geom.Envelope;
+
+import java.util.List;
+
+public class VehiclePresence {
+
+    private String vehicle;
+    private double cellLength;
+    private double cellWidth;
+    private String city;
+    private CityBound cityBounds;
+    private List<Prediction> predictions_15;
+    private List<Prediction> predictions_30;
+    private List<Prediction> predictions_45;
+    private long timestamp;
+
+    public long getTimestamp() {
+        return timestamp;
+    }
+
+    public void setTimestamp(long timestamp) {
+        this.timestamp = timestamp;
+    }
+
+    public String getVehicle() {
+        return vehicle;
+    }
+
+    public void setVehicle(String vehicle) {
+        this.vehicle = vehicle;
+    }
+
+    public double getCellLength() {
+        return cellLength;
+    }
+
+    public void setCellLength(double cellLength) {
+        this.cellLength = cellLength;
+    }
+
+    public double getCellWidth() {
+        return cellWidth;
+    }
+
+    public void setCellWidth(double cellWidth) {
+        this.cellWidth = cellWidth;
+    }
+
+    public String getCity() {
+        return city;
+    }
+
+    public void setCity(String city) {
+        this.city = city;
+    }
+
+    public CityBound getCityBounds() {
+        return cityBounds;
+    }
+
+    public void setCityBounds(CityBound cityBounds) {
+        this.cityBounds = cityBounds;
+    }
+
+    public List<Prediction> getPredictions_15() {
+        return predictions_15;
+    }
+
+    public void setPredictions_15(List<Prediction> predictions_15) {
+        this.predictions_15 = predictions_15;
+    }
+
+    public List<Prediction> getPredictions_30() {
+        return predictions_30;
+    }
+
+    public void setPredictions_30(List<Prediction> predictions_30) {
+        this.predictions_30 = predictions_30;
+    }
+
+    public List<Prediction> getPredictions_45() {
+        return predictions_45;
+    }
+
+    public void setPredictions_45(List<Prediction> predictions_45) {
+        this.predictions_45 = predictions_45;
+    }
+
+    public static class Prediction {
+        private double lon;
+        private double lat;
+        private double value;
+
+        @JsonIgnore
+        private Envelope envelope;
+
+        public Prediction() {
+
+        }
+
+        public Prediction(double lon, double lat, double value, Envelope envelope) {
+            this.lon = lon;
+            this.lat = lat;
+            this.value = value;
```

Comment:
Can you change "value" to something more expressive?

Reply:
what exactly? For me value in class Prediction is self explanatory...